R/subsetting.R

```diff
@@ -87,8 +109,23 @@ get_queries <- function(aoi, area_name, years, models, parameters, scenarios,
   ntime_hist <- grid_ref$ntime_historical
   ntime_model <- grid_ref$ntime_model
   
+  # Match coordinate systems
+  aoi <- sp::spTransform(aoi, sp::CRS(grid_ref$crs))
+  
   # Get relative index positions to full grid
-  index_pos <- get_aoi_indexes(aoi, grid_ref)
+  # slight buffering of extent allows us to handle points
+  if (class(aoi) == "SpatialPointsDataFrame") {
+    orig_crs <- raster::projection(aoi)
+    # buffering is only possible in projected coordinate systems
+    proj_aoi <- sp::spTransform(aoi, sp::CRS("+init=epsg:5070"))
+    extended_aoi <- rgeos::gBuffer(proj_aoi, width=.1)
```

Comment:
It's worth adding a comment to explain 1) why a width of 0.1 is valid and 2) what the units of 0.1 are (meters? decimal degrees?)

Reply:
This is a question for @WilliamsTravis